Gemfile

```diff
@@ -13,9 +13,11 @@ gem 'config', '~> 1.7'
 gem 'dor-services-client', '~> 1.2'
 gem 'honeybadger', '~> 4.1'
 gem 'jbuilder', '~> 2.5'
+gem 'lyber-core', '>= 4.0.3'
```

Comment:
Do you think we can inline the one function rather than pull in lyber-core?  Lyber-core is another dependency I'd like to kill off.